### PR TITLE
Add queue monitoring

### PR DESCRIPTION
**Pull request by lazedo**

### CODE REVIEW

File: libsofia-sip-ua/nta/nta.c [hunk 1]
```diff
@@ -6466,14 +6480,8 @@ static int nta_incoming_response_headers(nta_incoming_t *irq,
     clone = 1, sip->sip_call_id = sip_call_id_copy(home, irq->irq_call_id);
   if (!sip->sip_cseq)
     clone = 1, sip->sip_cseq = sip_cseq_copy(home, irq->irq_cseq);
-  if (!sip->sip_via) {
```
Comment (mjerris) [inline libsofia-sip-ua/nta/nta.c:6469]: this part of the change seems unrelated and should be in a separate patch with at least details of why we are making this change

**Review by mjerris — CHANGES_REQUESTED** [review]

File: libsofia-sip-ua/nta/nta.c (hunk 1 above)
Comment (lazedo) [reply libsofia-sip-ua/nta/nta.c:6469]: https://github.com/signalwire/freeswitch/issues/319

Comment (lazedo) [reply libsofia-sip-ua/nta/nta.c]: removed

**Comment by lazedo** [comment]
@mjerris updated as requested. can we move forward with this ?

**Review by andywolk — APPROVED** [review]
